packages/shared/src/styles/utilities.css

```diff
@@ -4,6 +4,7 @@
 
     &:focus-visible {
       box-shadow: 0 0 0 0.125rem var(--theme-focus);
+      border-radius: 15px;
```

Comment:
Just converting from `px` to `rem` 🥳 

```suggestion
      border-radius: 0.9375rem;
```

Reply:
Done! 👍